modules/apps/portal-workflow/portal-workflow-taglib/src/main/resources/META-INF/resources/workflow_status/css/main.scss

```diff
@@ -0,0 +1,20 @@
+.modal-full-screen {
+	display: flex;
+	justify-content: center;
+
+	.modal-content,
+	.modal-body {
+		width: 1050px;
```

Comment:
Maybe we should set different width/height depending on screen size. Modal gonna be responsible and will prevent bad behaviors.
What do you think?